api/src/wfl/api/spec.clj

```diff
@@ -15,8 +16,8 @@
 (s/def ::environment string?)
 (s/def ::finished inst?)
 (s/def ::input string?)
-(s/def ::input_bam string?)
-(s/def ::input_cram string?)
+(s/def ::input_bam (s/and string? #(str/ends-with? % ".bam")))
```

Comment:
think this just needs `str/ends-with?` (i.e. I don't think the `and` is necessary)

Reply:
will update!